src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -336,28 +272,79 @@ class BaseCourseCard extends Component {
     return null;
   };
 
-  renderCourseStartDate = () => {
-    const { startDate, mode } = this.props;
+  renderOrganizationName = () => {
+    const { orgName, mode } = this.props;
+
     const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
+    const course = isExecutiveEducation2UCourse ? 'Executive Education' : 'Course';
```

Comment:
nit: perhaps `courseTypeLabel` so it doesn't get confused for something like a course title when used below.